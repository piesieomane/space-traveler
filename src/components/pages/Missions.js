import Mission from './Mission';
import './mission.css';

const Missions = () => (
  <div className="mission_table">
    {' '}
    <table>
      <thead>
        <th>Mission</th>
        <th>Description</th>
        <th>Status</th>
        <th>Join</th>
      </thead>
      <tbody>
        <Mission />
      </tbody>
    </table>
  </div>
);
export default Missions;
