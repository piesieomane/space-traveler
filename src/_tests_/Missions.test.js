import React from 'react';
import Renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import Missions from '../components/pages/Missions';

it('renders correctly', () => {
  const tree = Renderer.create(
    <React.StrictMode>
      <Provider store={store}>
        <BrowserRouter>
          <Missions />
        </BrowserRouter>
      </Provider>
    </React.StrictMode>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
