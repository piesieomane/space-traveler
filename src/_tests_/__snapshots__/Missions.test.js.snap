// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mission_table"
>
   
  <table
    className="styled-table"
  >
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th>
          Description
        </th>
        <th>
          Status
        </th>
        <th>
          Join
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
